docs/public/static/diffs/expo-ios.diff

```diff
@@ -2,59 +2,33 @@ diff --git a/ios/MyApp/AppDelegate.h b/ios/MyApp/AppDelegate.h
 index ef1de86..d3d75b0 100644
 --- a/ios/MyApp/AppDelegate.h
 +++ b/ios/MyApp/AppDelegate.h
-@@ -1,7 +1,8 @@
- #import <React/RCTBridgeDelegate.h>
+@@ -1,6 +1,7 @@
+ #import <RCTAppDelegate.h>
 +#import <Expo/Expo.h>
  #import <UIKit/UIKit.h>
- 
--@interface AppDelegate : UIResponder <UIApplicationDelegate, RCTBridgeDelegate>
-+@interface AppDelegate : EXAppDelegateWrapper <UIApplicationDelegate, RCTBridgeDelegate>
- 
- @property (nonatomic, strong) UIWindow *window;
- 
-diff --git a/ios/MyApp/AppDelegate.mm b/ios/MyApp/AppDelegate.mm
-index da9f60a..0164063 100644
---- a/ios/MyApp/AppDelegate.mm
-+++ b/ios/MyApp/AppDelegate.mm
```

Comment:
@kudo, should we also include `AppDelegate.mm` changes? I initially removed this because there are no changes in the `git diff` for this file.

Reply:
@amandeepmittal yes, it's correct. from 0.71, we don't have to touch AppDelegate.mm

Reply:
Awesome! Thank you for reviewing this and sharing the details 😄